packages/manager/modules/sms/src/sms/options/smppParameter/smppParameter.html

```diff
@@ -0,0 +1,39 @@
+<section class="telecom-sms-options-smppParameter">
+    <header>
+        <tuc-section-back-link
+            data-tuc-section-back-link-to-state="sms.service.options"
+            data-tuc-section-back-link-title="{{ 'sms_section_back_link' | translate }}"
+        >
+        </tuc-section-back-link>
+        <h1 data-translate="sms_options_smpp_parameter_title"></h1>
+    </header>
+
+    <main role="main">
```

Comment:
i am not sure it's the right location to use `main` element.
`main` element must be used to display the main page content.

I guess here a simple div is more relevant.